views/register.html

```diff
@@ -28,15 +28,15 @@ <h1 class="govuk-label-wrapper">
                   Email
                 </label>
               </h1>
-              <input class="govuk-input" id="email" name="email" type="email">
+              <input class="govuk-input" id="email" name="email" type="email" placeholder="Companies House email address">
```

Comment:
Just something small from a GOV.UK styling and accessibility point of view - placeholder text should not be used as it will vanish as soon as the user starts typing and can cause issues with screen readers and browser webpage translations.
[GOV.UK Text Input guidelines](https://design-system.service.gov.uk/components/text-input/) look at the How it works section for more information.

I suggest a better/more accessible solution would be to change the already existing label fields from "Email" to "Companies House Email Address" and "Name" to "Full Name", this would make it clearer what needs to be entered without the drawbacks of the placeholder attribute. (Same comment applies to the name placeholder)